src/app/pages/administration/jobs/scheduled/cron-gen.scss

```diff
@@ -9,20 +9,72 @@ $prefix: ".chpl";
     }
 
     #{$prefix}-col-2 {
-        width: calc(100% / 6);
+        width: calc(100% / 3);
         float: left;
     }
 }
 
 #{$prefix}-form-inline {
     display: flex;
+    gap: 8px;
     flex-direction: row;
+    padding: 8px 0;
+    justify-content: flex-start;
+    align-items: baseline;
+    flex-wrap: wrap;
 }
 
 #{$prefix}-form-control {
     margin: 0 .2rem;
 }
 
+#{$prefix}-form-group {
+    padding: 8px 0;
+}
+
+#{$prefix}-form-check-label {
+    padding-left: .4rem;
+}
+
 #{$prefix}-form-check-label {
     padding-left: .4rem;
 }
+
+#{$prefix}-nav-link {
+    border: 1px solid #156dac;
+    background-color: #ffffff;
+    color: #156dac;
+    padding: 4px 8px;
+    text-transform: uppercase;
+}
+
+#{$prefix}-nav-link.DAY.c-tab {
+    border-radius: 4px 0 0 4px;
+}
+
+#{$prefix}-nav-link.YEAR.c-tab {
+    border-radius: 0 4px 4px 0;
+}
+
+#{$prefix}-nav-link:active {
+    background-color: #eeeeee;
+}
+
+#{$prefix}-nav-link:focus {
+    background-color: #eeeeee;
+}
+
+#{$prefix}-nav-link:hover {
+    background-color: #eeeeee;
+}
+
+.c-tabs {
```

Comment:
Should these have the `prefix` prefixes?

Reply:
No, when I inspect this page.. the prefix is not in front of these classes meaning we don't need to use the prefix.